src/pages/home/ReportScreen.tsx

```diff
@@ -733,29 +755,6 @@ function ReportScreen({
         Report.readNewestAction(report.reportID);
     }, [report]);
 
-    if (
-        (!isLinkedActionInaccessibleWhisper && isLinkedActionDeleted) ||
-        (shouldShowSkeleton &&
-            !reportMetadata.isLoadingInitialReportActions &&
-            reportActionIDFromRoute &&
-            sortedAllReportActions &&
-            sortedAllReportActions?.length > 0 &&
-            reportActions.length === 0 &&
-            !isLinkingToMessage)
-    ) {
-        return (
-            <BlockingView
-                icon={Illustrations.ToddBehindCloud}
-                iconWidth={variables.modalTopIconWidth}
-                iconHeight={variables.modalTopIconHeight}
-                title={translate('notFound.notHere')}
-                shouldShowLink
-                linkKey="notFound.noAccess"
-                onLinkPress={navigateToEndOfReport}
-            />
-        );
-    }
```

Comment:
Refactored it so we use FullPageNotFoundView below. It's already wrapped in ScreenWrapper too which adds safe inset for native platform.